From 11639dc20b6b4393eafd7b74c8ff8e9f81f94a1e Mon Sep 17 00:00:00 2001
From: Russell Webb <russell.webb@intel.com>
Date: Wed, 11 Apr 2012 17:21:29 -0700
Subject: Disable SetupWizard in eng builds

If ro.setupwizard.mode is set to OPTIONAL, it will still appear on
first boot.  This sets mode to DISABLED so it will not appear in an
eng build.

Change-Id: Ifa740959a8f17bf9c43ca9e0be32ee81057e1df1
---
 core/main.mk |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/core/main.mk b/core/main.mk
index e03801c..11fe3c7 100644
--- a/core/main.mk
+++ b/core/main.mk
@@ -345,7 +345,7 @@ ifneq ($(filter ro.setupwizard.mode=ENABLED, $(call collapse-pairs, $(ADDITIONAL
   # Don't require the setup wizard on eng builds
   ADDITIONAL_BUILD_PROPERTIES := $(filter-out ro.setupwizard.mode=%,\
           $(call collapse-pairs, $(ADDITIONAL_BUILD_PROPERTIES))) \
-          ro.setupwizard.mode=OPTIONAL
+          ro.setupwizard.mode=DISABLED
 endif
 endif
 
-- 
1.7.4.1

